src/components/layout/RightSection.vue

```diff
@@ -108,7 +108,7 @@ export default {
     currencyDropdownOptions() {
       return Object.keys(this.currencyRates.aeternity)
         .map((key) => ({
-          text: `${this.getFiatVal(this.currencyRates.aeternity[key])} ${key.toUpperCase()}`,
+          text: `~${this.getFiatVal(this.currencyRates.aeternity[key])} ${key.toUpperCase()}`,
```

Comment:
I think this ~ is very annoying we should probably remove it from everywhere. Can you revert that change. We have to discuss it first probably.